### PR TITLE
Add a basic json to yaml converter

### PR DESCRIPTION
**Pull request by lazka**
Nothing fancy, but has some nice defaults like preserving order and
indenting lists.

### CODE REVIEW

File: flatpak-json2yaml.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    parser.add_argument('json_file', type=str,
+                        help='The flatpak JSON file to convert')
+    parser.add_argument('-o', '--output', type=str, dest='out_file',
+                        help='The yaml target path')
+    args = parser.parse_args()
+
+    with open(args.json_file, "rb") as h:
+        out_file = args.out_file
+        if out_file is None:
+            out_file = os.path.splitext(args.json_file)[0] + '.yaml'
```
Comment (TingPing) [inline flatpak-json2yaml.py]: We've been using the `yml` file extension mostly. Might as well be consistent.

**Comment by TingPing** [comment]
A lot of flatpak json files use comments which are non-standard, it would be nice to not fail on those.

File: flatpak-json2yaml.py (hunk 1 above)
Comment (lazka) [reply flatpak-json2yaml.py]: ok

**Comment by lazka** [comment]
> A lot of flatpak json files use comments which are non-standard, it would be nice to not fail on those.

ok, I'll have a look

**Comment by lazka** [comment]
https://github.com/flathub/io.github.gillesdegottex.FMIT/blob/master/io.github.gillesdegottex.FMIT.json

```
./flatpak-json2yaml.py io.github.gillesdegottex.FMIT.json
Removed comment in line 14: '/* acs_alsa support */'
Removed comment in line 16: '/* acs_jack support */'
Removed comment in line 83: '/* acs_qt does not work on Linux */'
Removed comment in line 88: '/* must be done before calling qmake, or translations will not be installed (?) */'
```

Not sure if this is enough.. ?

**Pull request merged**